spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfiguration.java

```diff
@@ -134,4 +141,32 @@ public DataflowRdbmsInitializer dataflowRdbmsInitializer(DataSource dataSource,
 			return dataflowRdbmsInitializer;
 		}
 	}
+
+	@Configuration
+	@ConditionalOnExpression("#{'${spring.cloud.dataflow.features.tasksEnabled:true}'.equals('true') && '${spring.cloud.dataflow.composed.task.enabled:true}'.equals('true')}")
+	public static class ComposedConfiguration {
+
+		private static final String COMPOSED_TASK_RUNNER_NAME =
+				"task.composed-task-runner";
+
+		private static final String COMPOSED_TASK_RUNNER_URI_BASE =
+				"maven://org.springframework.cloud:spring-cloud-dataflow-composed-task:";
+
+		@Autowired
+		private DataSource dataSource;
+
+		@EventListener
+		public void handleContextRefresh(ContextRefreshedEvent event) {
```

Comment:
While I agree with you this was one of the original tenants that was discussed during our design strategy.  Please comment on this in the design document so we can adjust it there.   And then once we all come to agreement, we can change this in the PR or submit a new PR.